include/arch/arm/aarch64/arm_mmu.h

```diff
@@ -7,6 +7,14 @@
 #ifndef ZEPHYR_INCLUDE_ARCH_ARM64_MMU_ARM_MMU_H_
 #define ZEPHYR_INCLUDE_ARCH_ARM64_MMU_ARM_MMU_H_
 
+/* Helper macros to define memory region size */
+#define SZ_4K	0x1000
+#define SZ_1M	(256 * SZ_4K)
+#define SZ_2M	(2 * SZ_1M)
+#define SZ_4M	(2 * SZ_2M)
+#define SZ_512M (256 * SZ_2M)
+#define SZ_1G   (2 * SZ_512M)
```

Comment:
Ideally, these should have less generic names (e.g. `BLK_SZ_...`, `MEM_SZ_...`, `MEM_BLK_SZ_...`, `MMU_SZ_...`) to prevent potential name conflicts. I understand Linux also has these, but they are defined in `include/linux/sizes.h`, which is a generic header for size definitions.

Reply:
Also FYI, there are these macros in Zephyr:
https://github.com/zephyrproject-rtos/zephyr/blob/3dd2f610b00282c7c549c2981f532f1bc3a80b2b/include/sys/util.h#L212-L215

Reply:
I think these are good enough for our use. Do you suggest switching to that itself?

Reply:
Directly using macros from `include/sys/util.h` now.